### PR TITLE
Final presentation of Javascript Capstone project

### PR DESCRIPTION
**Pull request by 0sugo**
In this project we:

- [x] Fetched data from TVmaze API and displayed it on the homepage 
- [x] Added the functionalities of liking items and displaying the number of likes using involvement API.
- [x] Added the functionalities of displaying popup windows, adding and viewing comments, and displaying the number of comments using involvement API
- [x] We also implemented unit tests on likes and comments counts.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <a href="">
+          <img src="" alt="logo">
+        </a>
+      </div>
+      <div class="menu-links">
+        <a href="" class="menu-link" >
+          Movies<span id="shows-count"></span>
+        </a>
+        <a href="" class="menu-link">Schedule</a>
+      </div>
```
Comment (Shedrack-Sunday) [inline dist/index.html:23]: - [ ] You did some great work here 👏 . However, let's make it more user-friendly in other to have better acceptance by spacing the navigation headers and aligning them to be on par with the design given.

## Your view:
image

## Wireframe:
image

File: src/modules/commentCounter.test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    const index = document.getElementById('index');
+    const array = document.getElementById('array');
+
+    await counter(index, array);
+    ctr = `Comments ${array.childElementCount}`;
+  });
+
+  it('returns same number of comments', () => {
+    expect(ctr).toBe('Comments 3');
+  });
```
Comment (Shedrack-Sunday) [inline src/modules/commentCounter.test.js]: - [ ] You did well with the test to a point 👏 . However, i suggest you try to cover more edge cases that will occur apart from just counting the items. 
- For example, the scenario in which there are `0` items on the page (counter function should return 0)

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @0sugo  and @rosemutai   👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing of of the requirements,  However, there are a few you need to work on, to move to the next project. 💯





## Highlights 
- Nice README structure ✔️ 
- Very nice UI choice👏 
- Moderate use of modules for code files ✔️ 
  
# Required Changes ♻️

- [x] In the project requirements, you're meant test for your counter functions covering a couple of edge cases, but yours is limited. kindly expand those tests to make sure your app will behave as expected 👍 
- There is a little adjustment needed on the UI concerning your Nav.

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Shisui6 — APPROVED** [review]
Hi @0sugo and @rosemutai, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Nice use of the TV maze API ✅
- Nice design ✅
- Great collaboration and presentation ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
